chimp.js

```diff
@@ -0,0 +1,45 @@
+module.exports = {
+  path: "tests/spec",
+  browser: "chrome",
+  log: "debug",
+  debug: true,
+  sync: false,
+
+  webdriverio: {
```

Comment:
Most of this config should be under `desiredCapabilities` I have put together the following snippet that also optionally enables headless.
```
webdriverio: {
    baseUrl: process.env.BASE_URL || 'http://localhost:3000',
    desiredCapabilities: {
      bail: 1,
      browserName: "chrome",
      javascriptEnabled: true,
      maxInstances: 1,
      loggingPrefs: {
        browser: "ALL",
        driver: "ALL",
        server: "ALL"
      },
      chromeOptions: {
        args: [
          process.env.HEADLESS ? "--headless" : "--start-maximized",
          "--disable-gpu",
          "--window-size=1920,1080",
          "--no-sandbox",
          "--disable-extensions"
        ]
      },
      waitforTimeout: 5000,
      waitforInterval: 2000
    }
  },
```